src/app/components/FilterPopup/FilterPopup.jsx

```diff
@@ -237,7 +238,11 @@ class FilterPopup extends React.Component {
                   onFilterClick={this.onFilterClick}
                 />
 
-                <FieldsetDate />
+                <FieldsetDate
+                  legend="Date"
+                  filterId="date"
```

Comment:
Do you need this prop here? You pass it to the FieldsetDate component but you're not using it.